web/src/views/LoginPortal/LoginPortal.tsx

```diff
@@ -114,9 +118,10 @@ export default function () {
                 <ComponentOrLoading ready={firstFactorReady}>
                     <FirstFactorForm
                         disabled={firstFactorDisabled}
+                        rememberMe={props.rememberMe}
                         onAuthenticationStart={() => setFirstFactorDisabled(true)}
                         onAuthenticationFailure={() => setFirstFactorDisabled(false)}
-                        onAuthenticationSuccess={handleAuthSuccess} />
+                        onAuthenticationSuccess={handleAuthSuccess}/>
```

Comment:
@clems4ever was there specific linting utilised for the Typescript?
I noticed that fragments usually have spaces before they're closed @james-d-elliott, not that this is a real issue just should be consistent.

Reply:
Thanks, wasn't intentional. The space is a legacy thing that is not *technically* required but good practice to insert. It's just a typo from me removing things.